images/tier1/navigate-dark.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="18" width="18" viewBox="0 0 18 18">
```

Comment:
I didn't realize that either actually - figured they were all new ones, ah well.